kiota_abstractions/utils.py

```diff
@@ -0,0 +1,26 @@
+import importlib.util
+import sys
+
+
+def lazy_import(name):
+    """Lazily imports a python module given its absolute path
+
+    Args:
+        name (str): Absolute path to the module
+
+    Returns:
+        module: The module to be imported
+    """
+    spec = importlib.util.find_spec(name)
+
+    loader = importlib.util.LazyLoader(spec.loader)
```

Comment:
the case where the loader or module cannot be found should probably be handled as well.

Reply:
As long as the spec is found, any further errors with loader and module will be gracefully handled by `importlib`.